next/components/molecules/sections/general/OfficialBoardSection/OfficialBoardSection.tsx

```diff
@@ -0,0 +1,50 @@
+import type { ParsedOfficialBoardDocument } from '@backend/services/ginis'
+import { BasicSearchOld, DocumentCards, NoResultsFound } from '@bratislava/ui-bratislava'
+import { useTranslation } from 'next-i18next'
+import React, { useState } from 'react'
+import { useQuery } from 'react-query'
+
+const OfficialBoardSection = () => {
+  const [search, setSearch] = useState('')
+
+  const { t } = useTranslation('common')
+
+  // TODO: Refactor, prefetch and extract fetcher.
+  const { data } = useQuery({
+    queryKey: ['OfficialBoard', search],
+    queryFn: () =>
+      fetch(
+        `/api/ginis/documents-list?${new URLSearchParams({
+          search,
+        })}`,
+      ).then((r) => r.json() as Promise<ParsedOfficialBoardDocument[]>),
+    keepPreviousData: true,
+  })
+
+  return (
```

Comment:
Where was this code before? I don't see any code like this removed in this PR so maybe you left some unused code somewhere?

Reply:
From `next/pages/mesto-bratislava/transparentne-mesto/uradna-tabula.tsx`, I will migrate the pages first and then remove the old code in a new PR.